src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,97 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+RowLayout {
+    id: layout
+
+    property alias model: syncStatusModel
+
+    spacing: 0
+
+    NC.SyncStatusSummary {
+        id: syncStatusModel
+    }
+
+    Connections {
+        target: syncStatusModel
+        function onSyncingChanged() {
+            if (!syncStatusModel.syncing) {
+                syncIcon.rotation = 0;
```

Comment:
cannot this be achieved with a binding inside `syncIcon` component ?
```
rotation: syncStatusModel.syncing ? 0 : 0
```
or using a `Binding` component ?
```
Binding {
    target: syncIcon
    when: syncStatusModel.syncing
    property: 'rotation'
    value: 0
}
```